src/gui/accountstate.h

```diff
@@ -161,6 +162,32 @@ class AccountState : public QObject, public QSharedData
     ///Asks for user credentials
     void handleInvalidCredentials();
 
+    /** Returns the user status (online, dnd, away, offline, invisible)
+     *  https://gist.github.com/georgehrke/55a0412007f13be1551d1f9436a39675
+    */
+    QString status() const;
+
+    /** Returns the user status Message (emoji + text)
+    */
+    QString statusMessage() const;
+
+    /** Returns the user status icon url
+    */
+    QUrl statusIcon() const;
+
+    /** Returns the user status retrieved by the notificatons endpoint: dnd or online
+     *  https://github.com/nextcloud/desktop/issues/2318#issuecomment-680698429
+    */
+    QString notificationStatus() const;
+
+    /** Set new user status retrieved by the notificatons endpoint: dnd or online
+    */
+    void setNotificationStatus(const QString &status);
+
+    /** Fetch the user status Message (emoji + text)
```

Comment:
Just asking: is it really an accurate description? Don't we fetch the rest of the status information? Only the message?

Reply:
You are right, this method was something else before.